devices/ikea.js

```diff
@@ -560,6 +560,13 @@ module.exports = [
         extend: tradfriExtend.light_onoff_brightness(),
         meta: {turnsOffAtBrightness1: true},
     },
+    {
+        zigbeeModel: ['Pendant lamp WW'],
+        model: 'Pendant lamp WW',
```

Comment:
What model number is written on the box? I believe something starting with a `T`